environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentLoadBalancerService.java

```diff
@@ -43,10 +52,15 @@ public EnvironmentLoadBalancerService(
         this.loadBalancerEntitlementService = loadBalancerEntitlementService;
     }
 
-    public void updateLoadBalancerInEnvironmentAndStacks(EnvironmentDto environmentDto, EnvironmentLoadBalancerDto environmentLbDto) {
+    public EnvironmentLoadBalancerUpdateResponse updateLoadBalancerInEnvironmentAndStacks(EnvironmentDto environmentDto,
```

Comment:
could you refactor this method a bit? splitting it into smaller methods would improve readability

Reply:
I've refactored it. Let me know if it's more readable now.